channeldb/graph.go

```diff
@@ -3195,6 +3195,31 @@ func isZombieEdge(zombieIndex *bbolt.Bucket,
 	return true, pubKey1, pubKey2
 }
 
+// NumZombies returns the current number of zombie channels in the graph.
+func (c *ChannelGraph) NumZombies() (uint64, error) {
+	var numZombies uint64
+	err := c.db.View(func(tx *bbolt.Tx) error {
+		edges := tx.Bucket(edgeBucket)
+		if edges == nil {
+			return nil
```

Comment:
True but it’s still correct that there are 0 zombie channels

Reply:
What if there were zombie channels but we don't know because the bucket doesn't exist when it should? That would point to something having gone wrong with the database state. Not a big deal though given that the only use case of this at the moment is for the RPC.

Reply:
to qualify as a zombie it has to be in the index, how could we have zombies if the top-level bucket is missing?